### PR TITLE
add the parameter : --no-display-prompt

### PR DESCRIPTION
**Pull request by YannFollet**
add the parameter : --no-display-prompt , combine with --log-disable, it will display only the generated tokens

### CODE REVIEW

File: examples/main/main.cpp [hunk 1]
```diff
@@ -681,6 +683,8 @@ int main(int argc, char ** argv) {
         // reset color to default if there is no pending user input
         if (input_echo && (int) embd_inp.size() == n_consumed) {
             console::set_display(console::reset);
+            display = true;
+
```
Comment (ggerganov) [inline examples/main/main.cpp:687]:
```suggestion
            display = true;
```

Comment (YannFollet) [reply examples/main/main.cpp:687]: sorry  I don't understand, you mean remove line 687 ?

Comment (cebtenzzre) [reply examples/main/main.cpp:687]: remove the trailing whitespace.

Comment (YannFollet) [reply examples/main/main.cpp]: should be fixed

**Pull request merged**